scripts/setup_osx.sh

```diff
@@ -1,38 +1,267 @@
+  boost \
+  boost-python \
+  ffmpeg \
+  sdl2 \
+  swig \
+  openmpi
+
+# Leave a note in ~/.bash_profile for the added environment variables
+if [[ "${_arg_modify_bashprof}" = on ]]; then
+  echo -e "\n# Added by the garage installer" >> "${BASH_PROF}"
+fi
+
+# Set up MuJoCo
+if [[ ! -d "${HOME}"/.mujoco/mjpro150 ]]; then
+  mkdir "${HOME}"/.mujoco
+  MUJOCO_ZIP="$(mktemp -d)/mujoco.zip"
+  wget https://www.roboti.us/download/mjpro150_osx.zip -O "${MUJOCO_ZIP}"
+  unzip -u "${MUJOCO_ZIP}" -d "${HOME}"/.mujoco
+else
+  print_warning "MuJoCo is already installed"
+fi
+# Configure MuJoCo as a shared library
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${HOME}/.mujoco/mjpro150/bin"
+LD_LIB_ENV_VAR="LD_LIBRARY_PATH=\"\$LD_LIBRARY_PATH:${HOME}/.mujoco/mjpro150"
+LD_LIB_ENV_VAR="${LD_LIB_ENV_VAR}/bin\""
+if [[ "${_arg_modify_bashprof}" = on ]]; then
+  echo "export ${LD_LIB_ENV_VAR}" >> "${BASH_PROF}"
+fi
+
+# Set up conda
+hash conda 2>/dev/null || {
+  CONDA_INSTALLER="$(mktemp -d)/miniconda.sh"
+  wget https://repo.continuum.io/miniconda/Miniconda2-latest-MacOSX-x86_64.sh \
+    -O "${CONDA_INSTALLER}"
+  chmod u+x "${CONDA_INSTALLER}"
+  bash "${CONDA_INSTALLER}" -b -u
+  # Add conda to executable programs
+  CONDA_SH="${HOME}/miniconda2/etc/profile.d/conda.sh"
+  . "${CONDA_SH}"
+  if [[ "${_arg_modify_bashprof}" = on ]]; then
+    echo ". ${CONDA_SH}" >> "${BASH_PROF}"
+  fi
+}
+conda update -q -y conda
+
+# Create conda environment
 conda env create -f environment.yml
-conda env update
+if [[ "${?}" -ne 0 ]]; then
+  print_error "Error: conda environment could not be created"
+fi
+
+# Extras
+conda activate garage
+# Prevent pip from complaining about available upgrades
+pip install --upgrade pip
+# Fix Box2D install
+# See https://github.com/openai/gym/issues/100
+# See https://github.com/pybox2d/pybox2d/issues/82
+pip uninstall -y Box2D Box2D-kengz box2d-py
+PYBOX2D_DIR="$(mktemp -d)/pybox2d"
+git clone https://github.com/pybox2d/pybox2d "${PYBOX2D_DIR}"
+cd "${PYBOX2D_DIR}"
+python setup.py build
+python setup.py install
+python setup.py develop
+cd "${GARAGE_DIR}"
+# We need a MuJoCo key to import mujoco_py
+cp ${_arg_mjkey} "${HOME}"/.mujoco/mjkey.txt
+brew reinstall gcc@7 --without-multilib
+# Build mujoco-py and check is correctly installed
+python -c 'import mujoco_py'
+# Install git hooks
+pre-commit install -t pre-commit
+pre-commit install -t pre-push
+pre-commit install -t commit-msg
+# Install a virtualenv for the hooks
+pre-commit
+conda deactivate
+# Set personal configuration parameters
+cp garage/config_personal_template.py garage/config_personal.py
+
+# Add garage to python modules
+export PYTHONPATH="$PYTHONPATH:${GARAGE_DIR}"
+PYTHON_ENV_VAR="PYTHONPATH=\"\$PYTHONPATH:${GARAGE_DIR}\""
+if [[ "${_arg_modify_bashprof}" = on ]]; then
+  echo "export ${PYTHON_ENV_VAR}" >> "${BASH_PROF}"
+else
+  echo -e "\nRemember to export the following environment variables before" \
+    "running garage:"
+  echo "${LD_LIB_ENV_VAR}"
```

Comment:
You may want to include the full commands, e.g. "export $MY_FOO_VAR". If a user needs your help with this message, they probably also don't know what `export` does.